public/sw.js

```diff
@@ -12,7 +12,9 @@ self.addEventListener('install', event => {
 
 const fetchEventHandler = async event => {
   if (
-    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/.+\.webp$/.test(event.request.url)
+    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|ace\/(standard|ws))\/.+(\.jpg|\.png).webp$/.test(
```

Comment:
I am thinking about how there are some images with different URL types we may also need to put .webp on in other areas of the code, and then remove it here in cases of it not being supported:

Images like the podcast promo image for example like https://ichef.bbc.co.uk/images/ic/512x512/p0htdh6t.png which we want to be https://ichef.bbc.co.uk/images/ic/512x512/p0htdh6t.png.webp. This one wouldn't fall into this condition, and so we wouldn't remove .webp in the cases webp isn't supported.

Reply:
To resolve this issues, do we want to use the less specific pattern below
`/^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/.+\.webp$/`
or expand the URL checks with something along [this](https://github.com/bbc/simorgh/pull/11595/files#diff-52550de5c2974525a080b5738d3ac102090a1c226f660df5183015286b2129bbR18) lines to cover each
scenario we need to support?
The limitation presented by a less specific pattern is the likelihood of matching
`.webp` URLs that we aren't aware of.

Reply:
I have looked throught the codebase and I don't see webp being set anywhere else, and it shouldn't be .webp coming from Ares. Maybe this will change when we do some of the work in the BFF to add webp in there, but for now it seems that it will be ok to make the pattern less specific to catch all ichef urls that have webp added.

Reply:
Sounds good to me.
Thanks for looking looking into this concern. With this knowledge, we know we won't
encounter any surprises.
I'll change the pattern to only check for `.webp` at the end.

Reply:
I've changed this to `(news|images|ace\/(standard|ws))` to
- retain the rationale behind the specificity introduced to the matching pattern [here](https://github.com/bbc/simorgh/pull/11094/files#diff-98fd1d4c44ad3934ee242b1f070d235cc39f11e46738f28de66c6740182c4706R15)
- ensure we're able to keep out [these](https://github.com/bbc/simorgh/pull/11595/files#diff-24df8c7fb292da93f3f905c0d928a9c411f03ee6262d675436a64244aae1dcdcR82-R84) wrong paths in the URL.
- to ensure the regex pattern remains performant by being able to weed out unsupported URLs based on the image path.
Do let me know if that's okay.